modules/benchmark/R/check_if_list_of_pfts.R

```diff
@@ -0,0 +1,19 @@
+#' @title check_if_list_of_pfts
+#' Checks if format contains a variable named "plant_functional_type"
+#' @param observation_one a vector of plant fucntional types, or species
+#' @param observation_two anouther vector of plant fucntional types, or species
+#' @param custom_table a table that either maps two pft's to one anouther or maps custom species codes to bety id codes. 
+#' In the second case, must be passable to match_species_id. 
+#' @return \code{boolean} 
+#' @author Tempest McCabe
+check_if_list_of_pfts<-function(vars){
+  
+  if("plant_functional_type" %in% vars){
```

Comment:
FWIW, could be shortened to `if ("plant_functional_type" %in% vars || "species_name" %in% vars)` -- the `||` is a single-value `OR` statement (and `|` is a vectorized `OR`).

Reply:
An even more elegant and efficient way to do this is to vectorize it:

```r
if (any(c("plant_functional_type", "species_name") %in% vars)) ...
```